modules/i18n/src/translations/fr-FR/portals/console.ts

```diff
@@ -8112,6 +8112,9 @@ export const console: ConsoleNS = {
                                     removeAPIResource: "Supprimer la ressource API"
                                 }
                             },
+                            notes: {
+                                applicationRoles: "Seules les API et les étendues (autorisations) autorisées dans l'application sélectionnée(<1>{{applicationName}}</1>) seront répertoriées pour être sélectionnées."
```

Comment:
Can we improve this with the help of Docs team?

Reply:
Ack